.prow.yaml

```diff
@@ -114,3 +114,22 @@ presubmits:
               cpu: 4
             limits:
               memory: 2Gi
+
+  - name: pull-operating-system-manager-e2e
```

Comment:
I guess this name is unintentionally left from the previous implementation, I suggest renaming it to `pull-operating-system-manager-test` or something similar.